### PR TITLE
test: (e2e) added multi input core e2e tests

### PR DESCRIPTION
**Pull request by nikvalor**
#### Please provide a brief summary of this pull request.

added multi input core e2e tests

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: e2e/wdio/core/fixtures/appData/multi-input-contents.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    'BLUEBERRY',
+    'ORANGE'
+];
+export const fruitsArr4 = [
+    'Apple',
+    'Banana',
+    'Pineapple',
+    'Tomato'
+];
+export const Arr = [
```
Comment (Seamoo13) [inline e2e/wdio/core/fixtures/appData/multi-input-contents.ts:40]: Could you please give this array a more meaningful name?

File: e2e/wdio/core/pages/multi-input.po.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    root = '#page-content';
+
+    activeDropdownButtons = '[ng-reflect-disabled="false"] button[ng-reflect-glyph="value-help"]';
+    activeInputs = '.fd-multi-input-field [ng-reflect-disabled="false"] input';
+    disableInputs = 'div.is-disabled';
+    options = 'fd-checkbox.ng-untouched';
+    expandedDropdown = '.fd-list';
+    multiInputOptions = 'fd-multi-input[inputid="multiInput"] .fd-token__text span';
+    buttonShowAll = 'a.fd-link';
+    hiddenAddonButtonInputOptions =  'fd-multi-input[inputid="noAddonMultiInput1"] .fd-token__text span';
```
Comment (Seamoo13) [inline e2e/wdio/core/pages/multi-input.po.ts:15]: seems like there's an extra space between the equals sign and selector

File: e2e/wdio/core/tests/multi-input.e2e-spec.ts [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+    afterEach(() => {
+        refreshPage();
+        waitForPresent(activeDropdownButtons);
+    }, 1);
+
+    it('Check RTL/LTR orientation', () => {
+        multiInputPage.checkRtlSwitch();
+    });
+
+    it('Verify inputs should have placeholder', () => {
```
Comment (Seamoo13) [inline e2e/wdio/core/tests/multi-input.e2e-spec.ts:36]: Why are some test cases in describe blocks and others not? Maybe better to organize them all in the same pattern of describe -> it -> it -> it describe -> it -> it

File: e2e/wdio/core/tests/multi-input.e2e-spec.ts [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+        const activeInputsLength = getElementArrayLength(activeInputs);
+        for (let i = 0; i < activeInputsLength; i++) {
+            if (i === 8) {
```
Comment (Seamoo13) [inline e2e/wdio/core/tests/multi-input.e2e-spec.ts:39]: better not to use else if / else. 
if (i === 8) {
do this...
continue;
}
if (i === 9) {
do this...
continue;
}
do this

**Review by Seamoo13 — CHANGES_REQUESTED** [review]

File: e2e/wdio/core/tests/multi-input.e2e-spec.ts [hunk 5]
```diff
@@ -0,0 +1,259 @@
...
+            click(approveButton);
+            const inputOptionsLength = getElementArrayLength(mobileInputOptions);
+            for (let i = 0; i < inputOptionsLength - 8; i++) {
+                scrollIntoView(mobileInputOptions, i);
+                expect(getText(mobileInputOptions, i)).toBe(fruitsArr[i]);
+            }
+        });
+    });
+
+    it('verify Display Object Property', () => {
```
Comment (AntonOlkhovskyi) [inline e2e/wdio/core/tests/multi-input.e2e-spec.ts:142]: Could you please add a bit more specific descriptions for it's

**Review by AntonOlkhovskyi — CHANGES_REQUESTED** [review]

**Review by Seamoo13 — APPROVED** [review]

**Review by InnaAtanasova — APPROVED** [review]

**Review by AntonOlkhovskyi — APPROVED** [review]

**Pull request merged**